tests/functional/tokenstorage/test_simplejson_file.py

```diff
@@ -10,44 +10,44 @@
 
 
 @pytest.fixture
-def filename(tempdir):
-    return os.path.join(tempdir, "mydata.json")
+def json_file(tmp_path):
+    return tmp_path / "mydata.json"
 
 
-def test_file_does_not_exist(filename):
-    adapter = SimpleJSONFileAdapter(filename)
+def test_file_does_not_exist(json_file):
+    adapter = SimpleJSONFileAdapter(json_file)
     assert not adapter.file_exists()
 
 
-def test_file_exists(filename):
-    open(filename, "w").close()  # open and close to touch
-    adapter = SimpleJSONFileAdapter(filename)
+def test_file_exists(json_file):
+    open(json_file, "w").close()  # open and close to touch
```

Comment:
```suggestion
    json_file.touch()
```

Reply:
I thought I did this...? But yes! Yes x 100.